stories/apps/TableView.stories.js

```diff
@@ -0,0 +1,654 @@
+import React from "react";
+import TableView from "../../src/components/apps/listing/TableView";
+
+export const AppsTableViewTest = () => {
+    const logger = (message) => {
+        console.log(message);
+    };
+
+    const listing = {
+        total: 643,
+        apps: [
```

Comment:
Just a suggestion, if these are the same apps as what's in the `Listing` story, one thing you can do to save space is move these to a `AppMocks` file.  I started doing that with mock data that I needed in multiple places, or even just to keep the story files a little smaller.